### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by behrouz2115**
Signup and Signin when a user is logged out

### CODE REVIEW

File: src/pages/SignUp/services/SignUp.Hooks.js [hunk 1]
```diff
@@ -6,22 +6,26 @@ import { paths } from '../../../services/path';
...
-        const { error } = await supabase.auth.signUp({ email, password });
+        const { error } = await supabase.auth.signUp({ email, password, navbar });
         if (error) throw error;
         toast({
           status: 'success',
           title: 'Sign up success',
           description: 'Check your mailbox!',
         });
-        navigate(paths.signin);
+        navigate(paths.home);
```
Comment (horizon0708) [inline src/pages/SignUp/services/SignUp.Hooks.js:28]: These are not in the scope of the task. Please revert them back.

Comment (behrouz2115) [reply src/pages/SignUp/services/SignUp.Hooks.js:28]: I changed it

Comment (horizon0708) [reply src/pages/SignUp/services/SignUp.Hooks.js:28]: You haven't pushed the changes here. If you've done it, it shouldn't show on this PR.

**Comment by horizon0708** [comment]
Fixing it and merging it in.

**Pull request merged**